arm-servermanagement/2016-07-01-preview/servermanagement.json

```diff
@@ -0,0 +1,2234 @@
+{
```

Comment:
indentation 2 spaces.
If using VS  Ctrl + A and then Ctrl + K + D.
Make sure that tab are set to 2 spaces and are also replaced by spaces in Tools -> Options settings for json documents.